classes/04class/iac/artifacts/example2/vpc.tf

```diff
@@ -0,0 +1,125 @@
+resource "aws_vpc" "devops_academy" {
+  cidr_block = "192.168.0.0/16"
+  tags = {
+    Name = "DevopsAcademy"
+  }
+}
+resource "aws_subnet" "subnet_private_a" {
+  vpc_id            = "${aws_vpc.devops_academy.id}"
+  cidr_block        = "192.168.10.0/24"
```

Comment:
For the cidr might be nice to just mention that tf provides a function for calculating that. Last class we didn't mention this during the class and the students found out anyway.

https://www.terraform.io/docs/configuration/functions/cidrsubnet.html

Reply:
done